docs/source/getting_started/getting_started.md

```diff
@@ -423,7 +379,285 @@ curl -X POST "$URL/users/u1234/personalized_documents" \
+semantic representation of snippet with the `sub_id: 2` of the given document.
+
+```{note}
+Currently, the whole document used for the search is excluded from the search results, so, the result will not contain snippets from the same document.
+```
+
+For personalize semantic search with provided history snippet ids can also be used instead of
+document ids:
+
+```json
+"personalize": {
+    "exclude_seen": true,
+    "user": {
+         "history": [
+             {
+                 "id": { "document_id": "valid_doc_id1", "sub_id": 0 },
+                 "timestamp": "2000-05-14T20:22:50Z"
+             },
+             {
+                 "id": { "document_id": "valid_doc_id1", "sub_id": 1 },
+                 "timestamp": "2000-05-15T20:22:50Z"
+             }
+         ]
+    }
+}
+```
+
+When the history contains an interaction with a document which has multiple snippets, it will be
+processed similar to registering an interaction with each snippet of that document.
+
+`exclude_seen` always refers to whole documents.
+In the example above, even if there were a snippet with id `{ "document_id": "valid_doc_id1", "sub_id": 30 }` it would exclude the whole document `"valid_doc_id1"` no matter how many snippets it has.
+
+Please be aware that using semantic search based on a document id of a document with multiple snippets has some technical limitations. Mainly instead of searching based on the document as a whole it might search based on
+a single snippet of the document. This will change in the future but until then we recommend to always use the snippet id for now.
+
+## Personalization
+
+It is possible to register interactions with specific snippets instead of whole documents:
+
+```bash
+curl -X PATCH "$URL/users/u1234/interactions" \
+    --header "authorizationToken: $FRONTOFFICE_TOKEN" \
+    --header "Content-Type: application/json" \
+    --data '{
+        "documents": [
+            { "id": { "document_id": "effaffe", "sub_id": 1 } },
+            { "id": { "document_id": "effaffe", "sub_id": 2 } },
+            { "id": { "document_id": "affeffa", "sub_id": 1 } }
+        ]
+    }'
+```
+
+When registering a interaction with a document which has multiple snippets it will be
+processed similar to registering an interaction which each snippet of that document.
+
+# Document preprocessing
+
+## Automatic document splitting
+
+The back office ingestion API provides functionality to automatically split a document into
+multiple parts and create a snippet for each of this parts.
+
+The system uses Natural language processing (NLP) algorithms to split the document.
+
+This algorithm will be improved over time. This means a document ingested now and a equal document ingested
+in the future might have different splits. Additionally, not all NLP splitting algorithms are deterministic so we can't guarantee fully deterministic behavior even with the same algorithm.
```

Comment:
```suggestion
This algorithm will be improved over time. This means a document ingested now and an equal document ingested
in the future might have different splits. Additionally, not all NLP splitting algorithms are deterministic, so we can't guarantee fully deterministic behaviour even with the same algorithm.
```

We use UK spelling

Reply:
The is a question I have since a while:

Most IT things use US english.  Our code and API uses US english. So why do we use UK english for the getting started documentation?